code/game/gamemodes/endgame/endgame.dm

```diff
@@ -49,10 +48,6 @@
 	if(decay_rate && prob(decay_rate))
 		DecayTurf(T)
 
-// In the event you just need something called every tick, here it is
-/datum/universal_state/proc/process()
```

Comment:
"no longer needed"? There's no point in removing it and somebody else will add it back in later anyways so why?